core/src/main/java/org/moditect/commands/GenerateModuleInfo.java

```diff
@@ -232,11 +247,32 @@ private void updateModuleInfo(Map<String, Boolean> optionalityPerModule, ModuleD
         if ( moduleName != null ) {
             moduleDeclaration.setName( moduleName );
         }
+        
+        opensResources.stream().forEach(resourcePackage -> moduleDeclaration.getDirectives().add(
```

Comment:
I see no mention of this in the README. Could you add it?

Reply:
done